tab_faq.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Notes
+layout: col-sidebar
+tab: true
+order: 4
+tags: wrongsecrets
+---
+
+## FAQ's
+
+### What are the cautions to be taken while storing secrets?
+When it comes to storing secrets, such as passwords, private keys, and API keys, there are several cautions that you should take to ensure that the secrets are kept secure. Some of the key cautions to consider are:
+
+1. **Encryption:** When storing secrets, they should be encrypted. If an attacker gains access to the storage location, they can't read the secrets without the encryption key.
+
+
+2. **Access control:** Secret storage locations should be tightly controlled and storage locations should only be accessed by authorized personnel and trusted individuals.
+
+
+3. **Secure storage:** Keep secrets in a secure location that resists physical and logical attacks. Secure file systems, hardware security modules, password managers, or cloud storage are all examples.
+
+
+4. **Regular rotation:** Secrets should be rotated on a regular basis. This reduces the risk of compromise due to long-term exposure and makes it more difficult for attackers to use stolen secrets.
+
+
+5. **Strong passwords:** Passwords used to protect secrets should be strong and unique. They should not be reused across multiple accounts or systems. Make your passwords complex by adding numbers, symbols, and alternating the case of letters, not simple passwords like "Hello world" or "abcd1234". The password "SGF2ZSBhIG5pY2UgZGF5" is a great example.
+
+
+6. **Monitoring:** The storage location should be monitored for any suspicious activity, such as attempts to access the location outside of authorized times or from unauthorized locations.
```

Comment:
Let's change this, if you agree @bendehaan : let's make this more of a `getting started to play` instruction to simulate what the task was: explain the user they can navigate to one of our domains, try the simple challenge0 with password X, but also can try challenge31 witha more complicated password <password spoil>...

Reply:
@bendehaan Sir, can you please share your opinion ;)

Reply:
@commjoen Shall I make it Pictorial?

Reply:
Hi @puneeth072003 , no need to make it pictorial. I understood from @bendehaan that he will be working at it (at the end of) next week to think of  good text.

Reply:
@bendehaan can you please give me a quick review here.